block.py

```diff
@@ -270,9 +270,6 @@ def isdoor(b):
             # rotate lily pads randomly
             elif blocks[i] == 111:
                 param2[i] = random.randint(0,3)
-            # melon/pumpkin blocks
-            elif blocks[i] == 86 or blocks[i] == 103:
-                param2[i] = random.randint(0,23)
```

Comment:
Maybe I missed it, but, what was the reason this can be removed?

Reply:
Melons don't need to be randomly rotated since they don't have any face. Pumpkins do have a face in mcl2/minecraft, so the code interferes with conversion of facedirs.